pkg/server/providers.go

```
@@ -85,6 +86,26 @@ func (s *Server) registerProviders() error {
 
 	log.Info("Providers registered")
 
+	var allRequirementsMet bool
+
+	requirements, err := s.Provider.CheckRequirements()
+	if err != nil {
+		return err
+	}
+	for _, req := range *requirements {
+
+		if req.Met {
+			log.Infof("Requirement met : %s ", req.Reason)
+		} else {
+			allRequirementsMet = false
+			log.Warnf("Requirement not met.... : %s", req.Reason)
+
+		}
+	}
+	if !allRequirementsMet {
```

Remove this. As I said, we should throw an error, just log a warning